personnel.md

```diff
@@ -87,6 +87,7 @@ Feel free to contact the secretaries at info AT php-fig.org. For more informatio
 | Mark Railton ([@railto])              | 2018-01-28 - 2018-06-05 |
 | Margret Staples ([@dead_lugosi])      | 2017-11-12 - 2019-05-31 |
 | Ian Littman ([@iansltx])              | 2018-08-26 - 2020-01-25 |
+| Matteo Beccati ([@mbeccati])          | 2019-05-26 - 2021-05-30 |
```

Comment:
Why former secretary? ;-)

Reply:
Because I didn't pay enough attention, sorry for that.